users/views.py

```diff
@@ -31,6 +33,74 @@ def post(self, request):
                 name            = data["name"],
             )
 
-            return JsonResponse ({"MESSAGE":"SUCCESS"}, status = 201)
         except KeyError:
             return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
+
+        return JsonResponse ({"MESSAGE":"SUCCESS"}, status = 201)
+
+class HostView(View):
+    @users.utils.user_validator
+    def get(self, request):
+        try:
+            host = Host.objects.get(user_id = request.user.id)
+            result   = {
+                'id'            : host.id,
+                'user_id'       : host.user_id,
+                'nickname'      : host.nickname,
+                'profile_url'   : host.profile_url
+            }
+        except AttributeError :
+            return JsonResponse({'message': 'INVALID_USER'}, status=401)
+
+        except KeyError:
+            return JsonResponse({'message': 'KEY_ERROR'}, status=401)
+
+        except Host.DoesNotExist:
+            return JsonResponse({'message': 'INVALID_USER'}, status=401)
+
+        return JsonResponse(result, status=200) 
+    
+    @users.utils.user_validator      
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            user = User.objects.get(id = request.user.id)
+            if Host.objects.filter(user_id = user.id).exists() :
+                return JsonResponse({"MESSAGE": "DUPLE_USER"}, status=404)
+            Host.objects.create(
```

Comment:
if문 사이에 한줄 씩 띄워주시면 좋을 것 같아요~
그리고, `user = User.objects.get(id = request.user.id)`로 호출할 필요없이, filter()에서 한번만 조회하는게 더 효율적인 것 같네요~

```suggestion
            data = json.loads(request.body)
            
            if Host.objects.filter(user_id = request.user.id).exists() :
                return JsonResponse({"MESSAGE": "DUPLE_USER"}, status=404)
                
            Host.objects.create(
```